staging/src/k8s.io/apimachinery/pkg/types/namespacedname_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package types
+
+import (
+	"testing"
+)
+
+// tests MarshalLog
```

Comment:
My suggestion was to add tests to `staging/src/k8s.io/component-base/logs/json/json_test.go`. As it stands now, we don't have confirmation that JSON output will use `MarshalLog`, in particular because there is no type assertion.

When adding test cases there you also probably will need less boilerplate code.

Reply:
okay, done!